chall03/fulldemo.py

```diff
@@ -0,0 +1,45 @@
+#! /usr/local/bin/python3
+
+import sys
+
+def error():
+    print ("usage: " + sys.argv[0] + " <1-9 squared_rows...>")
+    exit(1)
+
+def looper(array):
+	num = len(array)
+	res = []
```

Comment:
If you've done the error handling before your solution function you don't need to initialize a variable and could print one by one the result found